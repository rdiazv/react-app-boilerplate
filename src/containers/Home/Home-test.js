import {Â React, mount } from 'helpers/test-helper'
import Home from 'containers/Home'

describe('Home', () => {
  it('should render', () => {
    const component = mount(<Home />)

    expect(component).toMatchSnapshot()
  })
})
