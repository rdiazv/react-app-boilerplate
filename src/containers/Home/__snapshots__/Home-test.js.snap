// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render 1`] = `
<Home>
  <div>
    <h1>
      Home
    </h1>
  </div>
</Home>
`;
